esmvaltool/cmorizers/data/cmor_config/ESACCI-WATERVAPOUR.yml

```diff
@@ -1,34 +1,35 @@
 ---
 # Common global attributes for Cmorizer output
 
-
-# Input
-# CDR-2
-filename: 'ESACCI-WATERVAPOUR-L3S-TCWV-*-05deg-{year}{month}-fv3.1.nc'
-# CDR-1
-# filename: 'dataset3_1/CDR-1/monthlies/ESACCI-WATERVAPOUR-L3?-TCWV-*-05deg-{year}{month}-fv3.1.nc'
 attributes:
   dataset_id: ESACCI-WATERVAPOUR
-  # CDR-2
-  version: 'CDR2-L3S-05deg_fv3.1'
-  # CDR-1
-  # version: 'CDR1-L3-05deg_fv3.1'
+  version: CDR2_L3-COMBI-05deg_fv3.1
```

Comment:
Regarding issue #3051 we should avoid having underscores in the version tag up to now. 
Sorry, just heard today about that issue. But we should change that the version to  `CDR2-L3-COMBI-05deg-fv3.1`.